docs/frameworks/symfony.md

```diff
@@ -6,90 +6,112 @@ introduction: Learn how to deploy serverless Symfony applications on AWS Lambda
+
+- Create a `serverless.yml` configuration file optimized for Symfony.
+- Add the `.serverless` folder to the `.gitignore` file.
+
+Otherwise, you can create the `serverless.yml` file manually at the root of the project, with the following configuration:
 
 ```yaml
-service: bref-demo-symfony
+# serverless.yml
+
+service: symfony
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
-    environment:
-        # Symfony environment variables
-        APP_ENV: prod
+  name: aws
+  # The AWS region in which to deploy (us-east-1 is the default)
+  region: us-east-1
+  # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+  stage: dev
+  runtime: provided.al2
+  environment:
+    # Symfony environment variables
+    APP_ENV: prod
 
 plugins:
-    - ./vendor/bref/bref
-
-package:
-    exclude:
-        - node_modules/**
-        - tests/**
-        - var/**
-    include:
-        - var/cache/prod/** # allows to deploy a pre-warmed container
+  - ./vendor/bref/bref
 
 functions:
-    website:
-        handler: public/index.php
-        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
-        layers:
-            - ${bref:layer.php-74-fpm}
-        events:
-            - httpApi: '*'
-    console:
-        handler: bin/console
-        timeout: 120 # in seconds
-        layers:
-            - ${bref:layer.php-74} # PHP
-            - ${bref:layer.console} # The "console" layer
+  # This function runs the Symfony website/API
+  web:
+    handler: public/index.php
+    timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+    layers:
+      - ${bref:layer.php-80-fpm}
+    events:
+      - httpApi: '*'
+  # This function let us run console commands in Lambda
+  console:
+    handler: bin/console
+    timeout: 120 # in seconds
+    layers:
+      - ${bref:layer.php-80} # PHP
+      - ${bref:layer.console} # The "console" layer
+
+package:
+  patterns:
+    # Excluded files and folders for deployment
+    - '!node_modules/**'
+    - '!tests/**'
+    - '!var/**'
+    # If you want to include files and folders that are part of excluded folders,
+    # add them at the end
+    - 'var/cache/prod/**'
 ```
 
-We still have a few modifications to do on the application to make it compatible with AWS Lambda.
+You still have a few modifications to do on the application to make it compatible with AWS Lambda.
 
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 2 new methods to the `Kernel` class:
+Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` you need to customize where the cache and logs are stored in
+the `src/Kernel.php` file. This is done by extending the default `Kernel` class create by Symfony with the `Kernel` class provide by the bridge:
```

Comment:
```suggestion
Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and logs are stored 
in the `src/Kernel.php` file. This is done automatically by the bridge, you just need to use the `BrefKernel` class provided by the bridge instead of the default `BaseKernel`
```